component/main.jsonnet

```diff
@@ -28,8 +28,205 @@ local net_pol = kube.NetworkPolicy('allow-from-waf-namespaces') {
   },
 };
 
+local rules_crd_gateway = {
+  apiGroups: [ 'apiextensions.k8s.io' ],
+  resources: [ 'customresourcedefinitions' ],
+  resourceNames: [
+    'backendtlspolicies.gateway.networking.k8s.io',
+    'gatewayclasses.gateway.networking.k8s.io',
+    'gateways.gateway.networking.k8s.io',
+    'grpcroutes.gateway.networking.k8s.io',
+    'httproutes.gateway.networking.k8s.io',
+    'referencegrants.gateway.networking.k8s.io',
+    'tcproutes.gateway.networking.k8s.io',
+    'tlsroutes.gateway.networking.k8s.io',
+    'udproutes.gateway.networking.k8s.io',
+  ],
+  verbs: [ 'get' ],
+};
```

Comment:
why do we need to aggregate `get crd XXX`? doesn't really make sense for non-cluster-wide permissions.

Reply:
I was mimicing what the olm version does. And in the HRM PoC I somehow had to set it as well.

Reply:
Removed.